composeApp/src/androidMain/kotlin/io/github/couchtracker/db/tmdb/Adapters.kt

```diff
@@ -0,0 +1,21 @@
+package io.github.couchtracker.db.tmdb
+
+import app.cash.sqldelight.ColumnAdapter
+import app.moviebase.tmdb.model.TmdbMovieDetail
+import app.moviebase.tmdb.model.TmdbReleaseDates
+import io.github.couchtracker.tmdb.TmdbLanguage
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+val tmdbLanguageAdapter = object : ColumnAdapter<TmdbLanguage, String> {
+    override fun decode(databaseValue: String) = TmdbLanguage.parse(databaseValue)
+    override fun encode(value: TmdbLanguage) = value.serialize()
+}
+val tmdbMovieDetailAdapter = object : ColumnAdapter<TmdbMovieDetail, String> {
+    override fun decode(databaseValue: String) = Json.decodeFromString<TmdbMovieDetail>(databaseValue)
```

Comment:
Warning: we are tying ourselves here with the serialization provided by the Kotlin TMDB library (and by extension TMDB APIs). While this is convenient, have you considered the possibility of future changes breaking our cache? For instance, if TMDB adds a required filed in their output, the library could make add it non-nullable and we will not be able to parse an old value stored in the DB.

What would be our path forward in that case? Simply parse stuff by hand? How would we detect that? Can we maybe add some very simple unit tests for these adapters?